EIPS/eip-5219.md

```diff
@@ -0,0 +1,72 @@
+---
+eip: 5219
+title: Decentralized HTTP
+description: Allows the sending of HTTP-like requests to smart contracts
+author: Pandapip1 (@Pandapip1)
+discussions-to: https://ethereum-magicians.org/t/pr-5219-decentralized-http/9907
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-07-10
+requires: 20, 137, 165, 4834
+---
+
+## Abstract
+
+Ethereum is the most-established blockchain for building decentralized applications (referred to as `DApp`s). Due to this, the Ethereum DApp ecosystem is very diverse. However, one issue that plagues DApps is the fact that they are not fully decentralized. Specifically, to interface a "decentralized" application, one first needs to access a *centralized* website containing the DApp's front-end code. This presents a few issues, discussed in the Motivation.
```

Comment:
The abstract currently describes a problem, not a solution.  The abstract should try to describe the solution and leave a description of the problem to the Motivation section.

Reply:
I think I probably got sidetracked and forgot to write about the solution. I'll write that now.